msteams-platform/tabs/how-to/tabs-in-sharepoint.md

```diff
@@ -1,17 +1,19 @@
 ---
-title: Adding a Microsoft Teams tab in SharePoint as an SPFx web part
+title: Add Teams tab to SharePoint
 author: laujan
 description: How to deploy your existing Teams tab to SharePoint as a SharePoint Framework web part.
 keywords: teams tabs sharepoint framework development
 ms.topic: conceptual
 ms.author: lajanuar
 ---
 
-# Adding a Microsoft Teams tab in SharePoint as an SPFx web part
+# Add Teams tab to SharePoint 
+
+You can get a rich integration experience between Microsoft Teams and SharePoint by adding a Microsoft Teams tab in SharePoint as an SPFx web part. This document guides you on how you to take a tab from a Microsoft Teams sample app and use it in SharePoint. 
 
 ## Rich integration between Teams and SharePoint
```

Comment:
The images under this section and next section have a bullet. Also, please check the alignment.

Reply:
Can you also rewrite the sections instead of cross-referencing the entire paragraph for Teams Tab in Sharepoint and Sharepoint Framework in Teams sections?